bitnami/etcd/Chart.yaml

```
@@ -25,4 +25,4 @@ name: etcd
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/etcd
   - https://coreos.com/etcd/
-version: 8.7.6
+version: 8.7.7
```

This is a new feature, Could you bump the minor version?

```suggestion
version: 8.8.0
```

---

In another PRs i bumped path version:
https://github.com/bitnami/charts/pull/15353
https://github.com/bitnami/charts/pull/15352
https://github.com/bitnami/charts/pull/15349

---

@fmulero it doesn’t matter to me which version to bump but for uniformity I bumped the patch version. Can you merge this PR?

---

Thanks @alexandrovas for let me know.

I'll share this case with the rest of the team, IMO and according to [semver](https://semver.org/) all of these PRs should have bumped the minor version.

Anyway, LGTM